_posts/2025-03-13-CPANSec-Retrospective-2024.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title:  "CPANSec retrospective 2024"
+date:   2025-03-13 00:23:17 +0200
+categories: cpansec update
+author: Thibault Duponchelle
+excerpt: Here is the CPANSec 2024 Retrospective
+---
+
+That was a big year for CPANSec. 
+
+If it gets a bit under the radar to you, please find some of the achievements listed in this retrospective!
+
+## Common Vulnerabilities and Exposures (CVEs)
+This year was the year where CPANSec installed a new mindset for Perl 
+and their relationship towards Common Vulnerabilities and Exposures (CVEs).
+
+Until then, very few CVEs were emitted for Perl modules and tooling. 
+And people and authors were a bit reluctant to deal with that process.
+
+CVE are not a sign of bad quality or maintenance of a code, 
+but a professionnal way of tagging and tracking a vulnerability, 
+also proving the importance of the concerned code.
+
+### Vulnerable Spreadsheet Parsing modules
+Start of the year was marked by vulnerabilities reported for [Spreadsheet::ParseExcel](https://metacpan.org/pod/Spreadsheet::ParseExcel).
```

Comment:
s/^/The /

Also, maybe add a few more words on the significance of this vulnerability, perhaps with a link to @timlegge's summary post? https://security.metacpan.org/2024/02/10/vulnerable-spreadsheet-parsing-modules.html

Reply:
Fixed for Start.

All links below are pointing to anchors of this page